dependencyLibs/Mavlink/src/com/MAVLink/Messages/ApmCommands.java

```diff
@@ -12,6 +12,7 @@
 	CMD_NAV_LOITER_TIME ("LoiterT",MAV_CMD.MAV_CMD_NAV_LOITER_TIME ,CommandType.NAVIGATION), /* Loiter around this MISSION for X seconds |Seconds (decimal)| Empty| Radius around MISSION, in meters. If positive loiter clockwise, else counter-clockwise| Desired yaw angle.| Latitude| Longitude| Altitude|  */
 	CMD_NAV_RETURN_TO_LAUNCH("RTL",MAV_CMD.MAV_CMD_NAV_RETURN_TO_LAUNCH,CommandType.COMMAND), /* Return to launch location |Empty| Empty| Empty| Empty| Empty| Empty| Empty|  */
 	CMD_NAV_LAND("Land",MAV_CMD.MAV_CMD_NAV_LAND,CommandType.NAVIGATION), /* Land at location |Empty| Empty| Empty| Desired yaw angle.| Latitude| Longitude| Altitude|  */
+	CMD_NAV_DO_LAND_START("Do Land Start",MAV_CMD.MAV_CMD_DO_LAND_START,CommandType.NAVIGATION), /* Mission command to prepare for a landing.  |Empty| Empty| Empty| Empty| Latitude| Longitude| Altitude|  */
```

Comment:
I don't think this class is being used by the project anymore. I removed it in [this commit](https://github.com/DroidPlanner/DroneKit-Android/commit/2a6438cf1b5dced6cb67c674f0211d6d10fa4879).